content/blog/2019-10-02-dev-guide-update-fall19.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: dev-guide-update-fall19
+title: 'rOpenSci Dev Guide 0.3.0: Updates Inside and Out'
+date: 2019-10-02
+authors:
+  - Scott Chamberlain
+  - Brooke Anderson
+  - Anna Krystalli
+  - Lincoln Mullen
+  - Karthik Ram
+  - Noam Ross
+  - Maëlle Salmon
+  - Melina Vidoni
+categories: blog
+topicid:
+tags:
+  - Software Peer Review
+---
+
+As [announced in February](/blog/2019/02/01/software-review-news/) updates to our Software Peer Review system, all our package development, review and maintenance is available as [an online book](https://devguide.ropensci.org/). Our goal is to update it approximately quarterly - it's time to present the third version. You can [read the changelog](https://devguide.ropensci.org/booknews.html) or this blog post to find out what's new in our dev guide 0.3.0!
+
+### For editors
+
+We've added a [book release guide](https://github.com/ropensci/dev_guide/issues/152) - a guide for shepherding a new version of the developer guide - which culminates in this blog post!
+
+### Guide tweaks
+
+The developer guide now lives at <https://devguide.ropensci.org/>!
+
+We fixed formatting of Appendices B-D in the pdf version of the book (see PDF button in the navbar) with [an issue](https://github.com/ropensci/dev_guide/issues/179) by [Indrajeet Patil](https://github.com/IndrajeetPatil).
+
+For book contributors, we've update [instructions in the README](https://github.com/ropensci/dev_guide#notes-for-associate-editors) for rendering the developer guide locally (<https://github.com/ropensci/dev_guide/issues/192>).
+
+[Hugo Gruson][hugog] helped us with a number of things: added a note for package maintainers about the [importance of GitHub recognizing your repository as the language R](https://github.com/ropensci/dev_guide/pull/172), and helped us [update links to `https` from `http`](https://github.com/ropensci/dev_guide/pull/167).
+
+
+
+### Updates to our policies and guidance
+
+#### Scope
+
+We've introduced an important change for anyone thinking of submitting a package. The former "reproducibility" category has been split up into four new categories (with "reproducibility" itself removed). They are:
+
```

Comment:
In the current website design, to make lists look ok I think one needs one empty line between items.

Reply:
will do